deepchecks/nlp/datasets/classification/just_dance_comment_analysis.py

```diff
@@ -28,26 +28,132 @@
 import pathlib
 import typing as t
 
+import numpy as np
 import pandas as pd
 
 from deepchecks.nlp import TextData
 from deepchecks.utils.builtin_datasets_utils import read_and_save_data
 
 __all__ = ['load_data']
 
-
 _FULL_DATA_URL = 'https://figshare.com/ndownloader/files/40564895'
-
+_SHORT_DATA_URL = 'https://figshare.com/ndownloader/files/40576232'
+_SHORT_PROPERTIES_URL = 'https://figshare.com/ndownloader/files/40580693'
+_SHORT_EMBEDDINGS_URL = 'https://figshare.com/ndownloader/files/40576328'
+_SHORT_PROBAS_URL = 'https://figshare.com/ndownloader/files/40578866'
 
 ASSETS_DIR = pathlib.Path(__file__).absolute().parent.parent / 'assets' / 'just_dance_comment_analysis'
 
 _METADATA_COLS = ['likes', 'dateComment']
 _CAT_METADATA = []
 _CAT_PROPERTIES = ['Language']
 _TEXT_COL = 'originalText'
+_TIME_COL = 'dateComment'
+_DATE_TO_SPLIT_BY = '2015-01-01'
+
+def load_precalculated_predictions(pred_format: str = 'predictions', as_train_test: bool = True,
+                                   use_full_size: bool = False) -> \
+        t.Union[np.array, t.Tuple[np.array, np.array]]:
+    """Load and return a precalculated predictions for the dataset.
+
+    Parameters
+    ----------
+    pred_format : str, default: 'predictions'
+        Represent the format of the returned value. Can be 'predictions' or 'probabilities'.
+        'predictions' will return the predicted class for each sample.
+        'probabilities' will return the predicted probabilities for each sample.
+    as_train_test : bool, default: True
+        If True, the returned data is split into train and test exactly like the toy model
+        was trained. The first return value is the train data and the second is the test data.
+        Otherwise, returns a single object.
+    use_full_size : bool, default: False
+        If True, the returned data will be the full dataset, otherwise returns a subset of the data.
+    Returns
+    -------
+    predictions : np.ndarray
+        The prediction of the data elements in the dataset.
+
+    """
+    if use_full_size:
+        raise NotImplementedError('Predictions for the full dataset are not yet available.')
```

Comment:
why is this parameter then even needed? maybe drop it?

Reply:
For compatibility with the other loading functions in this file I would imagine. 
@Nadav-Barak I would think though that the reasonable default for use_full_data is true.

Reply:
The partial size is good enough, 15K samples